lambda_functions/notification_service/src/item_closed_handler.py

```diff
@@ -0,0 +1,73 @@
+import logging
+
+from core_layer import helper
+from core_layer.db_handler import Session
+from core_layer.handler.notification_template_handler import S3NotificationTemplateHandler
+from core_layer.responses import BadRequest, InternalError, Success
+from core_layer.handler import item_handler
+from notification_service.src.sender.mail_sender import MailSender
+from notification_service.src.sender.telegram_sender import TelegramSender
+
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+notification_template_handler = S3NotificationTemplateHandler()
+
+mail_sender = MailSender(notification_template_handler)
+telegram_sender = TelegramSender(notification_template_handler)
+
+
+def handle_item_closed(event, context):
+    try:
+        helper.log_method_initiated("Send notification", event, logger)
+
+        if "item_id" not in event:
+            return BadRequest(event, "Event contains no item_id.")
+
+        with Session() as session:
+            item_id = event["item_id"]
+            item = item_handler.get_item_by_id(item_id, session)
+
+            if item is None:
+                return BadRequest(event,
+                                  f"No item was found with the given item_id [{item_id}].")
+
+            # TODO: This implementation is not ideal: 1.55 is rounded to 1.5. However, 1.56 is correctly rounded to 1.6.
+            rating = round(item.result_score, 1)
+            rating_text = get_rating_text(rating)
+
+            parameters = dict(
+                rating=rating, rating_text=rating_text, item_id=item.id, content=item.content
```

Comment:
I don't really understand where the actual message to send comes from. Shouldn't this line handover an `html` parameter to be replaced in the sender? 
Or am I misunderstanding something? 
Also I failed to find the line/file where the actual text/html, which is being sent is defined.

Reply:
Nevemind @SandroGlueck, I got it know.